elasticgraph-schema_artifacts/spec/unit/elastic_graph/schema_artifacts/runtime_metadata/enum_spec.rb

```diff
@@ -21,6 +21,71 @@ module Enum
 
             expect(enum_type).to eq Enum::Type.new(values_by_name: {})
           end
+
+          describe "#grouping_missing_value_placeholder" do
+            it "returns the MISSING_VALUE_PLACEHOLDER when there are no conflicting values" do
+              enum_type = Enum::Type.new(values_by_name: {
+                "active" => Enum::Value.new(
+                  sort_field: nil,
+                  datastore_value: "ACTIVE",
+                  datastore_abbreviation: nil,
+                  alternate_original_name: nil
+                ),
+                "inactive" => Enum::Value.new(
+                  sort_field: nil,
+                  datastore_value: "INACTIVE",
+                  datastore_abbreviation: nil,
+                  alternate_original_name: nil
+                )
+              })
```

Comment:
We can greatly trim these tests down if you define an `enum_value_with` factory method alongside `enum_type_with` in [runtime_metadata_support.rb](https://github.com/block/elasticgraph/blob/c1696deb24825c8488f29f0b0bb3ddf3afa17f0a/spec_support/lib/elastic_graph/spec_support/runtime_metadata_support.rb#L129). (I'm surprised there's not already an `enum_value_with` method there!). The method should provide benign defaults for all arguments so that tests can be written which only specify the details which matter to the test, helping to avoid excessive noise in the test (and reducing the amount of updates needed when a new required arg is added to one of those classes).